glui_control.cpp

```diff
@@ -313,12 +319,18 @@ int GLUI_Control::char_width(char c)
 { /* Hash table for faster character width lookups - JVK
        Speeds up the textbox a little bit.
   */
-  int hash_index = c % CHAR_WIDTH_HASH_SIZE;
+  /*Killing this with fire.
```

Comment:
We could make the cache static, but it would also need to be keyed on the GLUT font.  I just don't see much value is messing with this in isolation.